prac_06/guitar.py

```diff
@@ -0,0 +1,16 @@
+class Guitar:
+
+    def __init__(self, name="", year=0, cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        return "{guitar.name} {guitar.year} : ${guitar.cost:.2f}".format(guitar=self)
+
+    def get_age(self):
+        age = 2020 - self.year
```

Comment:
If you wont to keep the year up to date try:
import datetime
CURRENT_YEAR = datetime.date.today().year